strax/mailbox.py

```diff
@@ -164,12 +165,7 @@ def _send_from(self, iterable):
             self.kill(reason=e.args[0])
             # Do NOT raise! One traceback on the screen is enough.
         except Exception as e:
-            try:
-                e_args = e.args[0]
-            except IndexError:
-                e_args = ''
-            self.kill(reason=f"{e.__class__.__name__}('{e_args}') "
-                             f'in {threading.current_thread().name}')
+            self.kill(reason=(e.__class__, e, sys.exc_info()[2]))
```

Comment:
If I add variables to the exception (e.key), will that get propogated out?

Reply:
Yes, it reraises the original object